packages/odyssey-react-mui/src/labs/DataComponents/StackContent.tsx

```diff
@@ -0,0 +1,238 @@
+  currentLayout: StackLayout;
+  data: MRT_RowData[];
+  getRowId: UniversalProps["getRowId"];
+  stackOptions: StackProps;
+  isLoading: boolean;
+  isEmpty?: boolean;
+  isNoResults?: boolean;
+  hasRowReordering: UniversalProps["hasRowReordering"];
+  isRowReorderingDisabled?: boolean;
+  onReorderRows: UniversalProps["onReorderRows"];
+  totalRows: UniversalProps["totalRows"];
+  rowReorderingUtilities: {
+    dragHandleStyles: CSSObject;
+    dragHandleText: {
+      title: string;
+      "aria-label": string;
+    };
+    draggableTableBodyRowClassName: ({
+      currentRowId,
+      draggingRowId,
+      hoveredRowId,
+    }: {
+      currentRowId: string;
+      draggingRowId?: string;
+      hoveredRowId?: string;
+    }) => string | undefined;
+    handleDragHandleKeyDown: ({
+      table,
+      row,
+      event,
+    }: {
+      table: MRT_TableInstance<MRT_RowData>;
+      row: MRT_Row<MRT_RowData>;
+      event: React.KeyboardEvent<HTMLButtonElement>;
+    }) => void;
+    handleDragHandleOnDragCapture: (
+      table: MRT_TableInstance<MRT_RowData>,
+    ) => void;
+    handleDragHandleOnDragEnd: (table: MRT_TableInstance<MRT_RowData>) => void;
+    resetDraggingAndHoveredRow: (table: MRT_TableInstance<MRT_RowData>) => void;
+    updateRowOrder: ({
+      rowId,
+      newRowIndex,
+    }: {
+      rowId: string;
+      newRowIndex: number;
+    }) => void;
+  };
+  hasRowSelection: UniversalProps["hasRowSelection"];
+  rowSelection: MRT_RowSelectionState;
+  setRowSelection: Dispatch<SetStateAction<MRT_RowSelectionState>>;
+  emptyState: ReactNode;
+  pagination: {
+    pageIndex: number;
+    pageSize: number;
+  };
+  draggingRow?: MRT_Row<MRT_RowData> | null;
+};
+
+const StackContent = ({
+  currentLayout,
+  data,
+  stackOptions,
+  isLoading,
+  isEmpty,
+  isNoResults,
+  hasRowReordering,
+  isRowReorderingDisabled,
+  onReorderRows,
+  rowReorderingUtilities,
+  hasRowSelection,
+  rowSelection,
+  setRowSelection,
+  emptyState,
+  pagination,
+  totalRows,
+}: StackContentProps) => {
+  const handleRowSelectionChange = useCallback(
+    (row: MRT_RowData) => {
+      setRowSelection((prev) => {
+        const newSelection = { ...prev };
+        if (newSelection[row.id]) {
+          delete newSelection[row.id];
+        } else {
+          newSelection[row.id] = true;
+        }
+        return newSelection;
+      });
+    },
+    [setRowSelection],
+  );
+
+  const { updateRowOrder } = rowReorderingUtilities;
+
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+
+  const StackContainer = styled("div", {
+    shouldForwardProp: (prop) =>
+      prop !== "odysseyDesignTokens" &&
+      prop !== "currentLayout" &&
+      prop !== "maxGridColumns",
+  })<{
+    odysseyDesignTokens: DesignTokens;
+    currentLayout: StackLayout;
+    maxGridColumns: number;
+  }>(({ odysseyDesignTokens, currentLayout, maxGridColumns }) => ({
+    display: currentLayout === "stack" ? "flex" : "grid",
```

Comment:
If we are just stacking the children, this doesn't need to be `display: flex` right? Could just be `block`. We could lose the `flexDirection` declaration also if that is the case.